cpu/cc26xx_cc13xx/include/periph_cpu_common.h

```diff
@@ -65,6 +65,11 @@ typedef enum {
     GPIO_BOTH = IOCFG_EDGEDET_BOTH
 } gpio_flank_t;
 
+/**
+ * @brief   CPU specific GPIO pin generator macro
+ */
+#define GPIO_PIN(x, y)  (y)
```

Comment:
actually you don't need this - the fallback from `periph/gpio.h` should be used if this is not defined.

Reply:
>actually you don't need this - the fallback from periph/gpio.h should be used if this is not defined.

But, doesn't `periph/gpio.h` includes `periph_conf.h`? In order to use `GPIO_PIN` macro defined there on the `periph_conf.h` file

Reply:
Yea the easiest solution will be to just define `GPIO_PIN` in `periph_cpu_common.h` as you did to avoid the cyclic dependency.
But so something with the first argument so gcc is happy.

Reply:
Fixed, and rebased as well :-)